### PR TITLE
Work popover

### PR DESCRIPTION
**Pull request by IshmaelKargbo**
We worked on the:

- desktop popover window
- Mobile popover window 

please review and give your comment

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -318,7 +349,43 @@ button:focus {
...
+  padding: 16px;
+}
+
+.main-view .details {
+  display: flex;
+  flex-direction: column;
+}
+
+.project-model,
+.menu-modal {
```
Comment (VICTOR-CHUKWU) [inline style.css:382]: - [x] Nice job so far, But you just have a little fixing to make. Your modal desktop view is taking whole screen height making it difficult to see all the content in the modal, kindly fix this.
   how it looks 👇 
      
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @IshmaelKargbo  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (IshmaelKargbo) [reply style.css:382]: Thanks 👍😊

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @IshmaelKargbo, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**